addons/web_editor/static/src/js/backend/field_html.js

```diff
@@ -235,9 +235,9 @@ var FieldHtml = basic_fields.DebouncedField.extend(TranslatableFieldMixin, {
         this.$content.toggleClass('d-none');
         if ($codeview.hasClass('d-none')) {
             this.wysiwyg.odooEditor.observerActive();
-            this.wysiwyg.setValue($codeview.val());
+            this.wysiwyg.$editable.find('.o_editable')[0].innerHTML = $codeview[0].value;
```

Comment:
@dmo-odoo
Since the root of the codeview I have changed from the editable to the droppable zone, it is no longer possible to use the setValue() function with $codeview.val(). setValue() will assign the value of codeview to the innerHTML of editable instead of droppable zone and doing this will remove some styles and elements available before the dropable zone.

To avoid this issue, the codeview value is now assigned to the innerHTML of the dropable zone here instead.